tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
```

Comment:
Good. Once we have this NApp ready to be moved to `kytos-ng` org we'll hook it with Scrutinizer which will run the tox file, and then on each PR we'll have an automatic execution. Nice.